### PR TITLE
Add teamcity support

### PR DESCRIPTION
**Pull request by hoeggi**
I added support for automatic teamcity statistics and visualization so it's easier to track size across builds. It's configuarable and defaults to disabled.

### CODE REVIEW

File: src/main/kotlin/com/vanniktech/android/apk/size/ApkSizeExtension.kt [hunk 1]
```diff
@@ -5,4 +5,5 @@ open class ApkSizeExtension {
    * Set the max APK size in Kb. Greater than specified size will fail the build. Default is -1.
    */
   var maxApkSize = -1
+  var teamcity = false
```
Comment (vanniktech) [inline src/main/kotlin/com/vanniktech/android/apk/size/ApkSizeExtension.kt]: Can you add some documentation to this?

File: src/main/kotlin/com/vanniktech/android/apk/size/ApkSizeTask.kt [hunk 2]
```diff
@@ -20,6 +20,10 @@ open class ApkSizeTask : DefaultTask() {
 
     val fileEnding = apk.extension.toUpperCase(Locale.US)
     logger.log(LogLevel.LIFECYCLE, "Total $fileEnding Size in ${apk.name} in bytes: $apkSize (${ApkSizeTools.convertBytesToMegaBytes(apkSize)} mb)")
+    if (extension.teamcity) {
```
Comment (vanniktech) [inline src/main/kotlin/com/vanniktech/android/apk/size/ApkSizeTask.kt]: can you insert a new line above?

**Review by vanniktech — APPROVED** [review]
That's quite cool. Can you show me a screenshot how this looks like in Teamcity?

**Review by vanniktech — APPROVED** [review]
Cool 👍 

One last thing. Can we by default enable this flag if we know that we're running on Teamcity?

Is there any variable that we can use to set the value to true? Something like $TEAMCITY_VERSION or anything?

**Comment by hoeggi** [comment]
I updated the code and also added some dokumentaion in code and in the readme. 
There is an overview with a screenshot here https://confluence.jetbrains.com/display/TCD10/Statistic+Charts

Since I'm not using it long enough for my builds my graph looks kind of boring :)

I got the basic idea for this from the dexcount plugin here https://github.com/KeepSafe/dexcount-gradle-plugin and they both make for some nice longime reporting.

**Comment by hoeggi** [comment]
Not sure yet if it's possible to see if teamcity is detectable but i try to figure it out.

**Comment by hoeggi** [comment]
Ok i found it, according to this https://confluence.jetbrains.com/display/TCD10/Predefined+Build+Parameters
there should be a property teamcity.version exactly for this usecase. 

I see if i can make this work with this.

**Comment by vanniktech** [comment]
Can't we just use the environment variable? `TEAMCITY_VERSION`

**Review by vanniktech — APPROVED** [review]
Awesome. Thank you!

**Pull request merged**